src/main/resources/querydsl_checks_base.xml

```diff
@@ -90,6 +91,11 @@
   </module>
 
   <module name="SuppressWarningsFilter"/>
+  <module name="SuppressionCommentFilter">
+    <property name="offCommentFormat" value="CSOFF\: ([\w\|]+)"/>
+    <property name="onCommentFormat" value="CSON\: ([\w\|]+)"/>
```

Comment:
So the same as the normal CHEKSTYLE:OFF?
Something like `CHECKSTYLE:OFF: ([\w|]+)`?